package.json

```diff
@@ -12,12 +13,9 @@
     "cmake-js": "^3.5.0",
     "nan": "^2.7.0"
   },
-  "scripts": {
-    "build": "cmake-js build",
-    "rebuild": "cmake-js -l=silly rebuild",
-    "install": "cmake-js compile"
-  },
```

Comment:
The Readme.md is now out of sync. @bassosimone can you provide instructions on how to build this or ideally fixup the package.json to make it work like it used to